en_ID/pages/profiles/tiyan.md

```diff
@@ -0,0 +1,3 @@
+# Tiyan Attirmidzi
+
+Hi I'm [Tiyan Attirmidzi](https://github.com/tiyan-attirmidzi), you can call me Tiyan. I like software engineering because it is very cool! And my favorite food is fried noodles.
```

Comment:
Something more creative will be much more appreciated here 😄

Reply:
@empeje well, I'll revise it :sweat_smile: